cmd/app/settings.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2022-2025 Salesforce, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+
+	"github.com/slackapi/slack-cli/internal/cmdutil"
+	"github.com/slackapi/slack-cli/internal/config"
+	"github.com/slackapi/slack-cli/internal/prompts"
+	"github.com/slackapi/slack-cli/internal/shared"
+	"github.com/slackapi/slack-cli/internal/slackerror"
+	"github.com/slackapi/slack-cli/internal/slacktrace"
+	"github.com/slackapi/slack-cli/internal/style"
+	"github.com/spf13/cobra"
+)
+
+var settingsAppSelectPromptFunc = prompts.AppSelectPrompt
+
+func NewSettingsCommand(clients *shared.ClientFactory) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "settings [flags]",
+		Short: "Open app settings for configurations",
+		Long: strings.Join([]string{
+			"Open app settings to configure an application in a web browser.",
+			"",
+			"Discovering new features and customizing an app manifest can be done from this",
+			fmt.Sprintf("intuitive interface for apps with a \"%s\" manifest source.", config.ManifestSourceRemote.String()),
+			"",
+			"This command does not support apps deployed to Run on Slack infrastructure.",
+		}, "\n"),
+		Example: style.ExampleCommandsf([]style.ExampleCommand{
+			{
+				Meaning: "Open app settings for a prompted app",
+				Command: "app settings",
+			},
+			{
+				Meaning: "Open app settings for a specific app",
+				Command: "app settings --app A0123456789",
+			},
+		}),
+		Args: cobra.MaximumNArgs(0),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return appSettingsCommandPreRunE(clients, cmd, args)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return appSettingsCommandRunE(clients, cmd, args)
+		},
+	}
+	return cmd
+}
+
+// appSettingsCommandPreRunE determines if the command can be run in a project
+func appSettingsCommandPreRunE(clients *shared.ClientFactory, cmd *cobra.Command, args []string) error {
+	ctx := cmd.Context()
+	err := cmdutil.IsValidProjectDirectory(clients)
+	if err != nil {
+		return err
+	}
+	if clients.Config.ForceFlag {
+		return nil
+	}
```

Comment:
suggestion: It might be worth adding a comment here to help other maintainers know why we return early when the `--force` flag is provided 🤔 
```suggestion
	// Allow the force flag to ignore hosted apps and try to open app settings
	if clients.Config.ForceFlag {
		return nil
	}
```

Reply:
@mwbrooks Sounds good! I agree the placement is otherwise left to interpretation 📚 

I noticed too a test for this case is missing which doesn't seem ideal to me since this is not an erroring path that we support. Albeit the `--force` flag might make this more opinion, but let's avoid regressions with another test 🔍